corehq/apps/locations/util.py

```diff
@@ -27,6 +27,8 @@ def load_locs_json(domain, selected_loc_id=None, include_archived=False,
     * all top level locs
     * if a 'selected' loc is provided, that loc and its complete
       ancestry
+
+    show_admin - if True get all locations else get only administrative locations
```

Comment:
i'm finding this name confusing. can we make this input be `only_administrative` and then reverse the logic (so instead of `if not show_admin` it's `if only_administrative`?